infrastructure/src/main/java/org/corfudb/infrastructure/BaseServer.java

```diff
@@ -34,26 +34,27 @@ public boolean isServerReadyToHandleMsg(CorfuMsg msg) {
         return getState() == ServerState.READY;
     }
 
-    @Override
-    public ExecutorService getExecutor(CorfuMsgType corfuMsgType) {
-        return executor;
-    }
-
-    @Override
-    public List<ExecutorService> getExecutors() {
-        return Collections.singletonList(executor);
-    }
-
     public BaseServer(@Nonnull ServerContext context) {
         this.serverContext = context;
-        executor = Executors.newFixedThreadPool(serverContext.getBaseServerThreadCount(),
+        executor = Executors.newFixedThreadPool(serverContext.getConfiguration().getNumBaseServerThreads(),
                 new ServerThreadFactory("baseServer-", new ServerThreadFactory.ExceptionHandler()));
     }
 
-    /** Handler for the base server. */
+    /** HandlerMethod for the base server. */
     @Getter
-    private final CorfuMsgHandler handler =
-            CorfuMsgHandler.generateHandler(MethodHandles.lookup(), this);
+    private final HandlerMethods handler =
+            HandlerMethods.generateHandler(MethodHandles.lookup(), this);
+
+    @Override
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4662121)